### PR TITLE
Replacing msg flare with react tostify

### PR DESCRIPTION
**Pull request by vrackovic**
- Installed `react-toastify` npm package
- Supported api call catched errors
- Created notification store to support global notifications
- Notifications support static messages (in this case, after successfully creating, updating or deleting item of each page, notification is woken up and assigned message corresponding to previous action).
- Fixed bug on test page where not existing function is triggered

PR Resolves: #338, #340 and #341

### CODE REVIEW

File: web/src/main/frontend/src/base/stores/FlareStore.js [hunk 1]
```diff
@@ -1,70 +0,0 @@
-import {action, observable} from 'mobx';
-import {TYPE} from '../../components/ui/MsgFlare.js';
-import * as R from 'ramda';
```
Comment (iscekic) [inline web/src/main/frontend/src/base/stores/FlareStore.js]: Let's remove `ramda` as a dependency - we aren't using it enough. 😄

File: web/src/main/frontend/src/base/stores/QuestionStore.js [hunk 2]
```diff
@@ -54,7 +59,20 @@ class QuestionStore {
       ? questionApi.updateQuestion
       : questionApi.createNewQuestion;
 
-    return submit(question).then(() => this.fetchQuestions());
+    return submit(question).then(() => {
+      this.fetchQuestions();
+      setNotificationMessage(
+        question.id
```
Comment (iscekic) [inline web/src/main/frontend/src/base/stores/QuestionStore.js]: No need for 2 objects if the only difference is the message.

File: web/src/main/frontend/src/base/errorHandler.js [hunk 3]
```diff
@@ -1,27 +1,15 @@
...
- * By default, all errors derived from API calls (marked with 'apiCallError') are handled
- * as flare error notifications.
- * Other error objects are re-thrown.
- *
- * @param err Error object
- */
-export const handleApiError = err => {
-  if (err.apiCallError) {
-    FlareStore.error(err);
+export const setNotificationMessage = error => {
```
Comment (iscekic) [inline web/src/main/frontend/src/base/errorHandler.js]: `error` is a misleading name, how about `notification`?

Comment (vrackovic) [reply web/src/main/frontend/src/base/errorHandler.js]: I agree. Will be changed to `notification` then.

File: web/src/main/frontend/src/base/stores/FlareStore.js (hunk 1 above)
Comment (vrackovic) [reply web/src/main/frontend/src/base/stores/FlareStore.js]: Actually, we use its `keys`, `head`, `compose` and `replace` functions in some places. Should it be refactored, or should `ramda` be used in those places?

**Pull request merged**